wherehows-web/tests/helpers/acceptance-test-helper.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+/**
+ * Funtion used to pause the test
+ */
+export function pause(timeout) {
+  let promise = $.Deferred();
```

Comment:
Please let's use native Promises rather then jQuery's non compliant Deferreds. Also there's a `delay`  function you can import and reuse
```javascript
import { delay } from 'wherehows-web/utils/promise-delay';
```

Reply:
fixed